sirepo/cookie.py

```diff
@@ -28,6 +31,8 @@
 
 _SERIALIZER_SEP = ' '
 
+_SRCONTEXT_COOKIE_STATE_KEY = 'cookie_state'
```

Comment:
`_SRCONTEXT_KEY = __name__` is better: locally and globally unique. Also less problematic on module/package renames.